src/panel_gwalker/_gwalker.py

```diff
@@ -65,25 +194,63 @@ class GraphicWalker(ReactComponent):
         doc="""Optional extra Graphic Walker configuration. For example `{"i18nLang": "ja-JP"}`. See the
     [Graphic Walker API](https://github.com/Kanaries/graphic-walker#api) for more details."""
     )
+    renderer: Literal["GraphicWalker", "TableWalker", "GraphicRenderer", "PureRenderer"] = param.Selector(
```

Comment:
I'm in doubt what to call the different renderers. I've kept the `graphic-walker` naming convention. In [Pygwalker Streamlit app](https://github.com/kanaries/pygwalker-in-streamlit/blob/main/pygwalker_demo.py) its different. The terms are `explorer`, `viewer` and `chart`. One is not included.

I was also in doubt whether to create seperate classes for each of the renderers. I did not do it because the javascript code is easier to maintain in this way. And you can change renderer without loosing state.


https://github.com/user-attachments/assets/9d563bc4-7d4b-42d5-973e-2368bfdec1f2

Reply:
I will rename to `explorer`, `profiler`, `viewer` and `chart`. This is easier to remember and aligns nicely with the *clone methods* of the same names.

Reply:
done